cli/crud/import/import.go

```diff
@@ -0,0 +1,1214 @@
+/*
+	The batches are transmitted to the router in the form of a serialized json (struct t_batch).
+	Feedback from crud is obtained through this way also.
+
+	At the end of the program batches amount will be ⌈csvRecodrdsAmount / batchSize⌉, where csvRecodrdsAmount
```

Comment:
Here and below the maximum line length is 100.

Reply:
Done